spec/index.bs

```diff
@@ -2639,7 +2675,14 @@ namespace GPUTextureUsage {
                             1. Return a new [=invalid=] {{GPUTexture}}.
 
                     1. Let |t| be a new {{GPUTexture}} object.
-                    1. Set |t|.{{GPUTexture/[[descriptor]]}} to |descriptor|.
+                    1. Set |t|.{{GPUTexture/size}}.{{GPUExtent3DDict/width}} to |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=].
+                    1. Set |t|.{{GPUTexture/size}}.{{GPUExtent3DDict/height}} to |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=].
+                    1. Set |t|.{{GPUTexture/size}}.{{GPUExtent3DDict/depthOrArrayLayers}} to |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=].
+                    1. Set |t|.{{GPUTexture/mipLevelCount}} to |descriptor|.{{GPUTextureDescriptor/mipLevelCount}}.
+                    1. Set |t|.{{GPUTexture/sampleCount}} to |descriptor|.{{GPUTextureDescriptor/sampleCount}}.
+                    1. Set |t|.{{GPUTexture/dimension}} to |descriptor|.{{GPUTextureDescriptor/dimension}}.
+                    1. Set |t|.{{GPUTexture/format}} to |descriptor|.{{GPUTextureDescriptor/format}}.
+                    1. Set |t|.{{GPUTexture/usage}} to |descriptor|.{{GPUTextureDescriptor/usage}}.
```

Comment:
nit: Attributes are "getters", so we can define algorithm steps to define what happens when you access them. Those steps could just reach into `this.[[descriptor]]` to get the value.

(I don't think attributes inherently have any storage behind them which is how I ended up with the language "backed by an immutable internal slot of the same name" in a few places like https://gpuweb.github.io/gpuweb/#dom-gpuuncapturederrorevent-error . So if we don't have [[descriptor]] then I guess we should probably do something like that.)